sections/exchange/TradeCard/Charts/PriceChartCard.tsx

```diff
@@ -145,134 +144,63 @@ const ChartCard: FC<ChartCardProps> = ({
 							)}
 							{change != null && <ChangePercent value={change} />}
 						</>
-					) : (
-						<CurrencyLabel>{t('common.price')}</CurrencyLabel>
 					)}
 				</ChartHeaderTop>
 				{!isMarketClosed && (
-					<Actions {...{ alignRight }}>
-						{PERIOD_LABELS.map((period) => (
-							<StyledTextButton
-								key={period.value}
-								isActive={period.value === selectedPeriod.value}
-								onClick={(event) => {
-									setSelectedPeriod(period);
-									if (period.period !== Period.ONE_MONTH) {
-										setSelectedChartType(ChartType.AREA);
-									}
-								}}
-							>
-								{t(period.i18nLabel)}
-							</StyledTextButton>
-						))}
+					<Actions reverseChildren={alignRight}>
+						<PeriodSelector>
+							{PERIOD_LABELS.map((period) => (
+								<StyledTextButton
+									key={period.period}
+									isActive={period.period === selectedPeriod}
+									onClick={(event) => {
+										setSelectedPeriod(period.period);
+									}}
+								>
+									{t(period.i18nLabel)}
+								</StyledTextButton>
+							))}
+						</PeriodSelector>
+						{!(!isSUSD && selectedPeriod === Period.ONE_MONTH) ? null : (
```

Comment:
So you might have to update to the latest candlesticks from dev. But the current logic is to **always** show the candlestick option, but put the user on 1M time period if they click it. Which is something we should carry over here.

Reply:
Gotcha. Fixed.